_posts/2020-09-21-porting-namecoin-in-tor-browser-to-macos.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: "Porting Namecoin in Tor Browser to macOS"
+author: Jeremy Rand
+tags: [News]
+---
+
+Hot on the heels of [porting Namecoin in Tor Browser to Windows]({{ "/2020/08/28/porting-namecoin-in-tor-browser-to-windows.html" | relative_url }}), it was time to do the final remaining desktop OS, macOS.  This one was always going to present a challenge in the QA testing department, because I don't own any macOS devices.  On the other hand, I fully expected that the amount of code written for macOS support was going to be pretty minimal; in fact I expected that other than tweaking a few constants, macOS support was likely to be a subset of the code needed for Windows support.  But of course, our friend Murphy wasn't going to let that assumption stand.
```

Comment:
>I fully expected that the amount of code written for macOS support was going to be pretty minimal

I fully expected the amount of code written for macOS support to be pretty minimal

Reply:
This suggestion erases the future-ish tense for "written", so I'd prefer not to make this change.

Reply:
I understand. In that case, how about "I fully expected that the amount of code written for macOS support would be going to be pretty minimal"?